apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import AnalyticsApp from './AnalyticsApp';
+import { render, screen } from '@testing-library/react';
+import runReportResponseHasViews from '../../../../../lambda/public/sampleData/runReportResponseHasViews.json';
+import runReportResponseNoView from '../../../../../lambda/public/sampleData/runReportResponseNoViews.json';
```

Comment:
was not sure how best to do this. I could move this mock to the shared file that Patrick created so we do not have crazy relative imports?

Reply:
i commented on the above – i think this is only temporary so we shouldn't worry about it for now, but yeah this is gross if it were to stay like this  :)